client/swagger.yaml

```diff
@@ -393,7 +395,7 @@ definitions:
 
   Error:
     properties:
-      faultMessage:
+      fault_message:
```

Comment:
:100: Thanks for noticing this. Please submit your swagger changes to the firecracker project's [swagger file](https://github.com/firecracker-microvm/firecracker/blob/master/api_server/swagger/firecracker.yaml) as well!

Reply:
https://github.com/firecracker-microvm/firecracker/pull/713